src/ray/gcs/gcs_client/service_based_accessor.cc

```diff
@@ -127,6 +127,17 @@ Status ServiceBasedJobInfoAccessor::AsyncGetAll(
   return Status::OK();
 }
 
+Status ServiceBasedJobInfoAccessor::AsyncGetNextJobID(const ItemCallback<int> &callback) {
+  RAY_LOG(INFO) << "Getting next job id";
+  rpc::GetNextJobIDRequest request;
+  client_impl_->GetGcsRpcClient().GetNextJobID(
+      request, [callback](const Status &status, const rpc::GetNextJobIDReply &reply) {
+        callback(reply.job_id());
```

Comment:
Maybe we need to check the status first?

Reply:
ok